app/models/user.rb

```diff
@@ -28,6 +28,7 @@ class User < ActiveRecord::Base
                                                account_activations),
                              message: "%{value} is reserved." }
   validates :street_1, :city, :state, :zipcode, presence: true
+  validates :slug, uniqueness: true
```

Comment:
flash back to categorizations... hopefully slug gets to stay in for the long haul.